task/host-sp-comms/src/main.rs

```diff
@@ -622,19 +653,35 @@ impl ServerImpl {
                 Some(response)
             }
             HostToSp::HostBootFailure { .. } => {
-                // TODO what do we do in reaction to this? reboot?
+                // TODO what do we do in reaction to this? reboot? forward to
```

Comment:
Definitely don't reboot. The host might still have things to do here (dump state to disk, drop to a debugger prompt) and will send a separate reboot request when it's ready.

Reply:
:+1: I'll adjust this comment, thanks